packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1712,6 +1752,19 @@ export class ContainerRuntime
 		this.updateDocumentDirtyState(newState);
 	}
 
+	/**
+	 * Updates the runtime's IdCompressor with the stashed state present in the given op. This is a bit of a
+	 * hack and is unnecessarily expensive. As it stands, every locally stashed op (all ops that get stored in
+	 * the PendingStateManager) will store their serialized representation locally until ack'd. Upon receiving
+	 * this stashed state, the IdCompressor blindly deserializes to the stashed state and assumes the session.
+	 * Technically only the last stashed state is needed to do this correctly, but we would have to write some
+	 * more hacky code to modify the batch before it gets sent out.
+	 * @param content - An IdAllocationOp with "stashedState", which is a representation of un-ack'd local state.
+	 */
+	private async applyStashedIdAllocationOp(content: any) {
```

Comment:
Can the type of content be `IdCreationRange` or something else rather than any?

Reply:
I created a type `IdCreationRangeWithStashedState` to represent stashed ops